### ISSUE TITLE
Feature:  Linux desktop menu entry

### ISSUE THREAD

**Issue by kfedick** [inkle/inky#301]
A useful feature for Linux would be a desktop menu entry so Inky can be started from the desktop menu instead of the terminal.  This can be done with an executable .desktop file which can also be double clicked on using a file explorer.  I can share one that I have written if that is not against the rules.  I can also write a bash script that automatically generates the file and copies it to the directory where the start menu .desktop files are stored (so it shows up in the desktop menus) if anyone is interested.

**Comment by manuq** [comment]
If you install Inky from flathub.org, you will get a menu entry: https://flathub.org/apps/details/com.inklestudios.Inky

**Comment by lunarcloud** [comment]
"Instead of the terminal" - You can just double click the executable (as long as it has permission) from the file manager.
Because their main distribution is just the application files in a zip, you'll have to just make your own desktop menu entry, because desktop files have to use absolute paths - you can't ship a desktop file that says "launch from here".

**Comment by kfedick** [comment]
>
> "Instead of the terminal" - You can just double click the executable (as
> long as it has permission) from the file manager.
>

Yes but a searchable menu item is more convenient.  It's why I offered to
set it up.

Because their main distribution is just the application files in a zip,
> you'll have to just make your own desktop menu entry, because desktop files
> have to use absolute paths - you can't ship a desktop file that says
> "launch from here".


That is what the Bash script I had offered to share addressed.  It
generated the desktop file from the working directory it ran from as the
path so it would work regardless of where the files were stored.  For
convenience, another script could also have been made to remove the desktop
link again if needed as well.  However, since manuq had already provided a
desktop link through the flathub package, it is not needed.  Thanks for
that link by the way manuq.  I thought I had replied to your post but I see
now that I did not.

**Comment by kfedick** [comment]
A contained package would be very useful for some users.  I did have
trouble installing the pre-requisites and may have installed more of one
library than was required (I mentioned this in another post).  For now, the
git version is fine for my purposes though.
